client/src/components/table-components/Dealer.js

```diff
@@ -1,7 +1,110 @@
+import "../../App.css";
+import React, { useState, useEffect } from "react";
+import convertSecondstoTime from "../../utils/timeConversor.js";
 import "./styles/Dealer.css";
 
-function Dealer() {
-  return <div>Dealer</div>;
+export default function Dealer(props) {
+  return (
+    <div className='dealer'>
+      <Issue
+        onConfirmation={props.setIssue}
+        issue={props.issue}
+        // isDisabled={!props.seatStatus}
+      />
+
+      <Timer
+        duration={props.roundDuration}
+        // status={props.seatStatus}
+      />
+    </div>
+  );
 }
 
-export default Dealer;
+function Issue(props) {
+  const [formStatus, setFormStatus] = useState(false);
+  const [issue, setIssue] = useState("");
+
+  useEffect(() => {
+    if (formStatus && document.getElementById("issue-input")) {
```

Comment:
you could assign this element to a variable and then call it inside the if statement.
const issueInput = document.getElementById("issue-input");
...
issueInput.focus()